src/tape_drivers/ibm_tape.c

```diff
@@ -67,12 +67,6 @@ DRIVE_DENSITY_SUPPORT_MAP jaguar_drive_density[] = {
 	/* TS1170 */
 	{ DRIVE_GEN_JAG7,  TC_MP_JF, TC_DC_JAG7,    MEDIUM_PERFECT_MATCH },
 	{ DRIVE_GEN_JAG7,  TC_MP_JF, TC_DC_UNKNOWN, MEDIUM_PROBABLY_WRITABLE },
-	{ DRIVE_GEN_JAG7,  TC_MP_JE, TC_DC_JAG6,    MEDIUM_WRITABLE },
-	{ DRIVE_GEN_JAG7,  TC_MP_JE, TC_DC_UNKNOWN, MEDIUM_WRITABLE },
```

Comment:
Lines 70-75 are being removed completely, since when I first implemented Jag7 support, I received misinformation that Jag 6 tapes cartridges (JE, JV, JM) could be used in TS1170 tape drive, but that's not the case at all. Only the JF tape cartridge can be used in the TS1170 tape drive.

Reply:
I might make a code mis-reading. It looks good!